ArmPkg/Drivers/CpuDxe/MemoryAttribute.c

```diff
@@ -82,6 +82,12 @@ GetMemoryAttributes (
   EFI_STATUS  Status;
 
   if ((Length == 0) || (Attributes == NULL)) {
+    DEBUG ((
+      DEBUG_ERROR,
+      "%a: BaseAddress 0x%llx Length is zero or Attributes is NULL\n",

```

Comment:
Please include a hint in the message which of the two failure cases triggered the error.

Reply:
@ardbiesheuvel, I have updated this, please re-review.